Dockerfile

```diff
@@ -1,28 +1,25 @@
 FROM node:8.4-alpine
-RUN apk --no-cache update && \
-    apk --no-cache add python python3==3.6.1-r3 python3-dev==3.6.1-r3 py-pip ca-certificates groff less bash make jq curl wget g++ zip git openssh && \
+RUN apk --no-cache add python python3==3.6.1-r3 python3-dev==3.6.1-r3 py-pip ca-certificates groff less bash make jq curl wget g++ zip git openssh && \
     pip --no-cache-dir install awscli && \
-    update-ca-certificates && \
-    rm -rf /var/cache/apk/*
```

Comment:
dont we need this line to reduce image size?

Reply:
Using the `--no-cache` option prevents `apk` from ever storing cache to begin with.